admin/config/pm2_servers/test.json

```diff
@@ -0,0 +1,23 @@
+{
+    "apps": [{
+        "args": [
+            "../build/admin",
+            "-p 8083",
+            "--silent"
+        ],
+        "env": {
+            "NODE_ENV": "test",
+        },
+        "exec_mode": "fork",
+        "name": "bpm_admin-test",
+        "script": "./node_modules/.bin/http-server"
+    }, {
+        "args": [
+            "../e2e/fixtures/demo_fixtures.json",
+            "--port=3010",
+        ],
+        "exec_mode": "fork",
+        "name": "bpm_api-test",
+        "script": "./node_modules/.bin/json-server"
```

Comment:
I'm still wondering if this is a good thing to not be able to use the real API. If we don't, e2e testing is going to be painful as we'll have to mock all routes we'll want to test